Gemfile

```diff
@@ -1,3 +1,22 @@
 source 'https://rubygems.org'
+git_source(:toptal) { |repo| "https://github.com/#{repo}.git" }
 
 gemspec
+
+gem 'active_data', github: 'toptal/active_data', branch: 'master'
+
+gem 'activerecord', '>= 5.0', '< 7'
```

Comment:
@rewritten Why this change?

Having dev dependencies in gemspec has the benefit of being compatible with the rest of the ecosystem. E.g. Rubygems maps reverse deps for a gem, by analysing gemspec files.

Unless there is a valid reason for this, I would ask you to revert this change. Thanks. 😄

Reply:
Ok. But rubygems never uses development dependencies when resolving transitive deps. I don't know where else the development dependencies are used (except for rubygems website).

Reply:
For reference, Marc-André answer to the same topic in [What are the practical advantages of using `add_development_dependency` in .gemspec?](https://stackoverflow.com/a/62737144/384417)


> It's not clear there is any.
>
> In theory, RubyGems has the information necessary to run tests with add_development_dependency and test_files (see this question).
>
> Some believe that this should be outside the scope of RubyGems (see this huge thread).
>
> In practice, there is currently no real advantage, and the fact that RubyGems still installs test files by default is a drawback, as might be the lack of flexibility that Gemfile offers.